### ISSUE TITLE
Option to disable the CDN processing (webjars)

### ISSUE THREAD

**Issue by sebdotv** [sbt/sbt-rjs#45]
Properly disabling CDN usage (e.g. for security reasons) with webjars isn't possible at the moment: digesting is not taken into account (the config re-writing does not occur).

Corresponding thread: https://groups.google.com/d/msg/play-framework/N4f8Cz_vAMM/ArD7B2AWAlgJ

**Comment by ricardobarone** [comment]
+1

**Comment by elvenking** [comment]
+1

**Comment by rlamarche** [comment]
+1

**Comment by bicouy0** [comment]
+1

**Comment by DieterLi** [comment]
+1

**Comment by DieterLi** [comment]
I think rjs could be improved much smarter. For instance, actually, I have 2 home pages, one for the whole web site, another used as the accessing gate to the backend management. And I use 2 main config files for them each. During the build pipeline, rjs only treat the one in index page, and leave the one in backend index page as it is.

So open an setup parameter for such kind of situation could be much more helpful to the real cases.

Thanks for the hard work.

**Comment by dsanchez31** [comment]
+1

**Comment by victorjussiani** [comment]
+1

**Comment by pbelmann** [comment]
+1

**Comment by anton-fomin** [comment]
+1

**Comment by legacycode** [comment]
+1

**Comment by legacycode** [comment]
it seems to me that the following parameter disable the cdn generation. now my webjars (jquery, knockout, etc.) are icluded in my module files and are minified. it would be great if somebody can confirm this.

webJarCdns := Map()

**Comment by damirv** [comment]
@legacycode Indeed, you can disable cdn by adding `RjsKeys.webJarCdns := Map.empty` to your build.sbt (at least, this works for me)

**Comment by zltnrbn** [comment]
+1

**Comment by MipH** [comment]
Where is vote button?..
+1

**Comment by ignacioPL** [comment]
+1

**Comment by huntc** [comment]
PRs are more powerful than +1s. :-)
